update/Pos/Update/Logic/Global.hs

```diff
@@ -158,9 +160,11 @@ usVerifyBlocks ::
 usVerifyBlocks verifyAllIsKnown blocks =
     withUSLogger $
     reportUnexpectedError $
-    processRes <$> run (runExceptT $ mapM (verifyBlock verifyAllIsKnown) blocks)
+    processRes <$> run (runExceptT action)
   where
-    run :: forall m a . PollT (DBPoll m) a -> m (a, PollModifier)
```

Comment:
Why did you delete the type? It was rather helpful.

Reply:
Restored.